cliTool/index.ts

```diff
@@ -602,11 +667,131 @@ async function app(): Promise<any> {
+            },
+            {
+              type: "input",
+              name: "liqId",
+              message: "liqId",
+            },
+            {
+              type: "input",
+              name: "rewardToken",
+              message: "rewardToken",
+            },
+            {
+              type: "input",
+              name: "tokenValue",
+              message: "tokenValue",
+            },
+          ])
+          .then(
+            async (answers: {
+              user: string;
+              liqId: string;
+              rewardToken: string;
+              tokenValue: string;
+            }) => {
+              await addActivatedLiquidityFor3rdPartyRewards(
+                new BN(answers.liqId.toString()),
+                new BN(answers.rewardToken.toString()),
+                new BN(answers.tokenValue.toString()),
+                answers.user,
+              );
+              return app();
+            },
+          );
+      }
+      if (answers.option.includes("Add activated native rewards liquidity")) {
+        return inquirer
+          .prompt([
+            {
+              type: "input",
+              name: "user",
+              message: "user",
+              default: "//Alice",
+            },
+            {
+              type: "input",
+              name: "liqId",
+              message: "liqId",
+            },
+            {
+              type: "input",
+              name: "tokenValue",
+              message: "tokenValue",
+            },
+          ])
+          .then(
+            async (answers: {
+              user: string;
+              liqId: string;
+              rewardToken: string;
+              tokenValue: string;
+            }) => {
+              await addActivatedLiquidityForNativeRewards(
+                new BN(answers.liqId.toString()),
+                new BN(answers.tokenValue.toString()),
+                answers.user,
+              );
+              return app();
+            },
+          );
+      }
+      if (answers.option.includes("Staked liq that is not activated")) {
+        return inquirer
+          .prompt([
+            {
+              type: "input",
+              name: "liqId",
+              message: "liquidity token ID",
+              default: "1",
+            },
+          ])
+          .then(async (answers: { liqId: number }) => {
+            const liqIdBn = toNumber(answers.liqId.toString());
+            await addStakedUnactivatedReserves(liqIdBn);
+            return app();
+          });
+      }
+      if (answers.option.includes("Add vesting tokens and move these to MPL")) {
+        return inquirer
+          .prompt([
+            {
+              type: "input",
+              name: "user",
+              message: "user",
+              default: "//Alice",
+            },
+            {
+              type: "input",
+              name: "tokenId",
+              message: "liquidity token ID",
+              default: "1",
```

Comment:
what 1 means? perhaps we can explain that of 1 -> create a pool in the message ?

Reply:
Yes, added